README.md

```diff
@@ -1,17 +1,38 @@
-# rpm-ostree Overview
-
-rpm-ostree is a hybrid image/package system.  It uses
-[OSTree](https://ostree.readthedocs.io/en/latest/) as a base image
-format, and supports RPM on both the client and server side using
-[libdnf](https://github.com/rpm-software-management/libdnf).
+# rpm-ostree: A true hybrid image/package system
+
+```
+                         +-----------------------------------------+
+                         |                                         |
+                         |       rpm-ostree (daemon + CLI)         |
+                  +------>                                         <---------+
+                  |      |     status, upgrade, rollback,          |         |
+                  |      |     pkg layering, initramfs --enable    |         |
+                  |      |                                         |         |
+                  |      +-----------------------------------------+         |
+                  |                                                          |
+                  |                                                          |
+                  |                                                          |
++-----------------|-------------------------+        +-----------------------|-----------------+
+|                                           |        |                                         |
+|         libostree (image system)          |        |            libdnf (pkg system)          |
+|                                           |        |                                         |
+|   C API, hardlink fs trees, system repo,  |        |    ties together libsolv (SAT solver)   |
+|   commits, atomic bootloader swap         |        |    with librepo (RPM repo downloads)    |
+|                                           |        |                                         |
++-------------------------------------------+        +-----------------------------------------+
+```
+
+rpm-ostree combines [libostree](https://ostree.readthedocs.io/en/latest/) (an image system),
+with [libdnf](https://github.com/rpm-software-management/libdnf) (a package system), bringing
+many of the benefits of both together.
 
 For more information, see the online manual: [Read The Docs (rpm-ostree)](https://rpm-ostree.readthedocs.org/en/latest/)
 
 **Features:**
 
- - Atomic upgrades and rollback for host system updates
- - A server side tool to consume RPMs and commit them to an OSTree repository
- - A system daemon to consume OSTree commits as updates
+ - Transactional upgrades and rollback for host system updates
```

Comment:
My small nit is that this doesn't capture the "versioning/image-based" part of it. Though I know we're trying to condense things here. How about: "Transactional **versioned** upgrades and rollback for host system updates"?

Reply:
OK I took a shot at that :arrow_down: